### ISSUE TITLE
MicroMagick + GraphicsMagick falls down on this image

### ISSUE THREAD

**Issue by dukedorje** [mceachen/micro_magick#5]
http://dreamcatcher.net/images/uploads/originals/393.JPG

`gm identify -verbose Sculpture-005.JPG`
Works, although it shows an incredibly large Maker Note.

```ruby
MicroMagick::Image.new("Sculpture-005.JPG").width
```
Hangs indefinitely.

Using ImageMagick instead of GraphicsMagick, this same command works.

**Comment by dukedorje** [comment]
Looks like MicroMagick falls down with both GraphicsMagick and ImageMagick on this file:

http://dreamcatcher.net/images/uploads/originals/5817.gif

```bash
> identify -verbose 5817.gif | wc -c
  638200
```
This has got to be overloading ruby's memory.  Is it assigning the output to a variable and doing a regex or something?

**Comment by mceachen** [comment]
I've added the JPG to the test suite and completely changed how the `identify` parsing works. This is fixed in v1.0.0, which I just released. 

Thanks for raising the issue!

**Comment by dukedorje** [comment]
:+1:
